examples/active-active/variables.tf

```diff
@@ -0,0 +1,53 @@
+variable "aws_role_arn" {
```

Comment:
Part of the task's Acceptance Criteria is Consistent alphabetization for variables (Grouping of variables to their relevant sections), so this file needs to be updated.

You may want to double-check the Acceptance Criteria one more time, too, before making these ready for review, just in case.

Reply:
sorry, these were fixed in other PR. Let me pull the latest changes here